### DISCUSSION TITLE
Theme file (thmutil) did not reconize my WixLocalization string

### DISCUSSION THREAD

**Discussion by osamax2** [wixtoolset/issues#7644] [Questions] [answered]
## Bugs

If this issue is a bug:

* Which version of WiX are you building with?
4.0.1+6e6eb478

* Which version of Visual Studio are you building with (if any)?
2022

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?
4

* Which version of .NET are you building with?
6

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?
10

* Describe the problem and the steps to reproduce it.
1. bootstrapper have a theme xml that needs to read my WixLocalization string
2. try to add to Editbox StringId like in [website](https://wixtoolset.org/docs/schema/thmutil/editbox/)
`<Editbox` Name="InstallFolder" X="11" Y="143" Width="-91" Height="21" TabStop="yes" FontId="3" FileSystemAutoComplete="yes" StringId="DefaultPath" `/>`
4. in my product.en-us.wxl
`<?xml` version="1.0" encoding="utf-8"?> 
`<WixLocalization` Culture="en-us" Language="1033" xmlns="http://wixtoolset.org/schemas/v4/wxl">.
	`<String` Id="DefaultPath" Value="C:\Program Files\" />
`</WixLocalization>`
5. No error or warning after build, just the installer does not open

* Describe the behavior you expected and how it differed from the actual behavior.
1. after build the installer work fine and i see the default path in Editbox as text

**Comment by barnson** [comment, marked as answer]
> StringId (xs:nonNegativeInteger) : Identifier that references a **string resource in the module** to define the initial text for the control.

StringId is the name of a string resource, not a localization string id.

**Comment by osamax2** [comment]
And what is String resource 
And how I make a default value to editbox

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [GitHub discussions area][gh]. You might also consider asking your question on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[so]: https://stackoverflow.com/questions/tagged/wix
